### ISSUE TITLE
Custom modules are not updating on signal

### ISSUE THREAD

**Issue by protecto** [Alexays/Waybar#289]
I am using Sway 1.0 with Waybar v0.6.0.

With the following config file:

```json
{
    "layer": "top",
    "height": 30,
    "modules-right": ["custom/foo"],
    "custom/foo": {
        "exec": "date",
        "interval": "once",
        "signal": 8
    }
}
```

After starting waybar and running `pkill -RTMIN+8 waybar` the time does not update.

Is there a problem with Waybar or am I doing something wrong?

**Comment by Alexays** [comment]
My bad it's fixed with e8f31a0c4f659f9ab36eae048b4ef927c6013db3

**Comment by kRHYME7** [comment]
Issue is back? anyone?
